steps/id-and-residency/id.tsx

```diff
@@ -88,10 +88,11 @@ const step: ProcessStepDefinition<ResidentDatabaseSchema, "id"> = {
           props: {
             name: "id-type",
             legend: (
-              <FieldsetLegend>What type of ID?</FieldsetLegend>
+              <FieldsetLegend>What type of ID</FieldsetLegend>
```

Comment:
```suggestion
              <FieldsetLegend>What type of ID?</FieldsetLegend>
```

That `?` belongs there.